Source/AlchemyLanguageV1/AlchemyLanguageV1.swift

```diff
@@ -0,0 +1,1613 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Alamofire
+import Freddy
+import RestKit
+
+public class AlchemyLanguageV1 {
+    
+    private let apiKey: String
+    
+    private let serviceUrl = "https://gateway-a.watsonplatform.net/calls"
+    private let errorDomain = "com.watsonplatform.alchemyLanguage"
+ 
+    let unreservedCharacters = NSCharacterSet(charactersInString: "abcdefghijklmnopqrstuvwxyz" +
+        "ABCDEFGHIJKLMNOPQRSTUVWXYZ" +
+        "1234567890-._~")
+    
+    public init(apiKey: String) {
+        self.apiKey = apiKey
+    }
+    
+    private func dataToError(data: NSData) -> NSError? {
+        do {
+            let json = try JSON(data: data)
+            let status = try json.string("status")
+            let statusInfo = try json.string("statusInfo")
+            let userInfo = [
+                NSLocalizedFailureReasonErrorKey: status,
+                NSLocalizedDescriptionKey: statusInfo
+            ]
+            return NSError(domain: errorDomain, code: 400, userInfo: userInfo)
+        } catch {
+            return nil
+        }
+    }
+    
+    private func shouldHaveField(field: Bool) -> String {
```

Comment:
This removes the tri-state functionality and forces a value.  Basically, this will have to match all of the default values of the service and it will be very easy to get out of sync with server